LanguageFeatures/Enhanced-Enum/implementing_enum_A04_t24.dart

```diff
@@ -18,9 +18,9 @@
 /// "primitive equality", and we want to ensure that enums can be used in
 /// switches.
 ///
-/// @description Check that it's not a compile-time error if a mixin declaration
-/// has Enum as a superinterface, and it declares a non-abstract instance member
-/// named `hashCode=`, but the setter/getter signature constraint does apply.
+/// @description Check that it's not an error if a mixin declaration has `Enum`
+/// as a superinterface, and it declares a non-abstract instance setter named
+/// `hashCode=`, but the setter/getter signature constraint does apply.
```

Comment:
I think this PR is all about removing references to that setter/getter constraint? Or is that some other rule? If so then we should probably just have this:
```suggestion
/// `hashCode=`.
```
This also matches the fact that there are no expected errors in this library now.

Reply:
Thanks! Updated here and in `implementing_enum_A04_t20.dart`